src/nft/components/collection/CollectionPageLoading.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from 'nft/components/Box'
```

Comment:
Can this be called `Skeleton.tsx`? We named the TokenDetails skeleton like this, and I'd like to be consistent across the codebase.

Reply:
Renamed to `CollectionPageSkeleton` to match `TokenDetailsPageSkeleton`